cns/requestcontroller/kubecontroller/crdtranslator.go

```diff
@@ -84,3 +87,22 @@ func CNSToCRDSpec(toBeDeletedSecondaryIPConfigs map[string]cns.SecondaryIPConfig
 
 	return spec, nil
 }
+
+func GetIPAddressStringAndPrefix(ipSubnetString string) (string, int, error) {
+	splitIPSubnet := strings.Split(ipSubnetString, "/")
+
+	if len(splitIPSubnet) != 2 {
+		err := fmt.Errorf(fmt.Sprintf("Failed to split IpSubnet string: [%s]", ipSubnetString))
```

Comment:
`fmt.Errorf` already allows for a format and arguments (the `f` suffix is a convention for this). No need for `fmt.Sprintf`